packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -382,8 +367,9 @@ class PerformanceMonitoring {
           lastSpan.subType === span.subType &&
           lastSpan.action === span.action &&
           lastSpan.name === span.name &&
-          span.duration() / transDuration < threshold &&
-          (span._start - lastSpan._end) / transDuration < threshold
+          span.duration() / transDuration < SIMILAR_SPAN_THRESHOLD_PERCENTAGE &&
+          (span._start - lastSpan._end) / transDuration <
+            SIMILAR_SPAN_THRESHOLD_PERCENTAGE
```

Comment:
I don't see a lot of benefit in using the constant here, instead of in the caller function. 
Furthermore, with a function argument we can test this function more thoroughly, i.e. with different ratios.

Reply:
Basically to avoid the validation part inside functions and reducing bundle size bytes. We can add the args back since it would be interesting to test as you mentioned, but will remove the validation since we know its a constant and its a super generic function to test for all possible cases.